context/2022/acr-wcag-21-sc.json-ld

```diff
@@ -0,0 +1,336 @@
+  "WCAG20:headings-and-labels": { "@type": "earl:TestRequirement" },
+  "WCAG20:focus-visible": { "@type": "earl:TestRequirement" },
+  "WCAG20:language-of-page": { "@type": "earl:TestRequirement" },
+  "WCAG20:language-of-parts": { "@type": "earl:TestRequirement" },
+  "WCAG20:on-focus": { "@type": "earl:TestRequirement" },
+  "WCAG20:on-input": { "@type": "earl:TestRequirement" },
+  "WCAG20:consistent-navigation": { "@type": "earl:TestRequirement" },
+  "WCAG20:consistent-identification": { "@type": "earl:TestRequirement" },
+  "WCAG20:error-identification": { "@type": "earl:TestRequirement" },
+  "WCAG20:labels-or-instructions": { "@type": "earl:TestRequirement" },
+  "WCAG20:error-suggestion": { "@type": "earl:TestRequirement" },
+  "WCAG20:error-prevention-legal-financial-data": { "@type": "earl:TestRequirement" },
+  "WCAG20:parsing": { "@type": "earl:TestRequirement" },
+  "WCAG20:name-role-value": { "@type": "earl:TestRequirement" },
+	"WCAG21:orientation": { "@type": "earl:TestRequirement" },
+	"WCAG21:identify-input-purpose": { "@type": "earl:TestRequirement" },
+	"WCAG21:reflow": { "@type": "earl:TestRequirement" },
+	"WCAG21:non-text-contrast": { "@type": "earl:TestRequirement" },
+	"WCAG21:text-spacing": { "@type": "earl:TestRequirement" },
+	"WCAG21:content-on-hover-or-focus": { "@type": "earl:TestRequirement" },
+	"WCAG21:character-key-shortcuts": { "@type": "earl:TestRequirement" },
+	"WCAG21:pointer-gestures": { "@type": "earl:TestRequirement" },
+	"WCAG21:pointer-cancellation": { "@type": "earl:TestRequirement" },
+	"WCAG21:label-in-name": { "@type": "earl:TestRequirement" },
+	"WCAG21:motion-actuation": { "@type": "earl:TestRequirement" },
+	"WCAG21:status-messages": { "@type": "earl:TestRequirement" },
+	"WCAG22:accessible-authentication": { "@type": "earl:TestRequirement" },
+	"WCAG22:dragging-movements": { "@type": "earl:TestRequirement" },
+	"WCAG22:consistent-help": { "@type": "earl:TestRequirement" },
+	"WCAG22:page-break-navigation": { "@type": "earl:TestRequirement" },
+	"WCAG22:focus-appearance-minimum": { "@type": "earl:TestRequirement" },
+	"WCAG22:visible-controls": { "@type": "earl:TestRequirement" },
+	"WCAG22:target-size-minimum": { "@type": "earl:TestRequirement" },
+	"WCAG22:redundant-entry": { "@type": "earl:TestRequirement" },
+
+	"WCAG21:identify-purpose": { "@type": "earl:TestRequirement" },
+	"WCAG21:timeouts": { "@type": "earl:TestRequirement" },
+	"WCAG21:animation-from-interactions": { "@type": "earl:TestRequirement" },
+	"WCAG21:target-size": { "@type": "earl:TestRequirement" },
+	"WCAG21:concurrent-input-mechanisms": { "@type": "earl:TestRequirement" },
+	"WCAG22:accessible-authentication-no-exception": { "@type": "earl:TestRequirement" },
+	"WCAG22:focus-appearance-enhanced": { "@type": "earl:TestRequirement" },
+	
+	"WCAG21Tech":"https://www.w3.org/WAI/WCAG21/Techniques/",
+	"WCAG22Tech":"https://www.w3.org/WAI/WCAG22/Techniques/",
+	
+  "WCAG21Tech:html/H44": { "@type": "earl:TestCase" },
```

Comment:
There is some addtional techniques. Voici tout les techniques: https://www.w3.org/WAI/WCAG22/Techniques/